src/lv2_plugin/CMakeLists.txt

```diff
@@ -45,6 +45,10 @@ configure_file(resources/manifest.ttl ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/rnnoise.
 configure_file(resources/rnnoise.ttl ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/rnnoise.lv2/rnnoise.ttl)
 
 install(TARGETS ${LV2_TARGET}
-        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/lv2
-        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}/lv2
-        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}/lv2)
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/lv2/rnnoise.lv2/
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}/lv2/rnnoise.lv2/
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}/lv2/rnnoise.lv2/)
+install(DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/rnnoise.lv2/
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/lv2/rnnoise.lv2/
+        FILES_MATCHING PATTERN "*.ttl"
+        )
```

Comment:
It's not immediately clear to me why this one shouldn't have a trailing slash, but the output is nicer this way (without affecting if it works):
```suggestion
install(TARGETS ${LV2_TARGET}
        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/lv2/rnnoise.lv2/
        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}/lv2/rnnoise.lv2/
        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}/lv2/rnnoise.lv2/)
install(DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/rnnoise.lv2/
        DESTINATION ${CMAKE_INSTALL_LIBDIR}/lv2/rnnoise.lv2
        FILES_MATCHING PATTERN "*.ttl"
        )

```

Before:
```
Install the project...
-- Install configuration: ""
-- Installing: /usr/local/lib/lv2/rnnoise.lv2/librnnoise_lv2.so
-- Up-to-date: /usr/local/lib/lv2/rnnoise.lv2/
-- Installing: /usr/local/lib/lv2/rnnoise.lv2//manifest.ttl
-- Installing: /usr/local/lib/lv2/rnnoise.lv2//rnnoise.ttl
-- Up-to-date: /usr/local/lib/ladspa/librnnoise_ladspa.so
```

After:
```
Install the project...
-- Install configuration: ""
-- Up-to-date: /usr/local/lib/lv2/rnnoise.lv2/librnnoise_lv2.so
-- Up-to-date: /usr/local/lib/lv2/rnnoise.lv2
-- Up-to-date: /usr/local/lib/lv2/rnnoise.lv2/manifest.ttl
-- Up-to-date: /usr/local/lib/lv2/rnnoise.lv2/rnnoise.ttl
-- Up-to-date: /usr/local/lib/ladspa/librnnoise_ladspa.so
```

Reply:
Is now fixed. I squashed it with my other commit.